### PR TITLE
Support Linux builds.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: examples/lwjgl/script/build.sh [hunk 1]
```diff
@@ -1,10 +1,17 @@
-#!/bin/zsh -euo pipefail
+#!/usr/bin/env -S /bin/zsh -euo pipefail
```
Comment (tonsky) [inline examples/lwjgl/script/build.sh:1]: Why do you need env here?

File: examples/lwjgl/script/build.sh [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 cd `dirname $0`/..
 
 mkdir -p target
 
 LWJGL_VER=3.2.3
 LWJGL_OS=macos
+LWJGL_OS=macos
```
Comment (tonsky) [inline examples/lwjgl/script/build.sh:8]: This line (and previous one) could be removed

File: examples/lwjgl/script/run.sh [hunk 3]
```diff
@@ -34,4 +40,4 @@ if [[ "$HAS_LWJLG" == "false" ]]; then
     mvn compile
 fi
 
-java -cp $CLASSPATH -XstartOnFirstThread -Djava.awt.headless=true -ea org.jetbrains.skija.examples.lwjgl.Main
\ No newline at end of file
+java -cp $CLASSPATH -Djava.awt.headless=true -ea org.jetbrains.skija.examples.lwjgl.Main
```
Comment (tonsky) [inline examples/lwjgl/script/run.sh:43]: `-XstartOnFirstThread` is mandatory on macOS

File: script/skia_linux.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env -S /bin/zsh -euo pipefail
+
+cd `dirname $0`/..
+
+cd third_party/skia
+git apply ../skia_skparagraph.patch || true
+gn gen out/Release-x64 --args="is_debug=false is_official_build=true skia_use_system_expat=false skia_use_system_icu=false skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_sfntly=false skia_use_freetype=true skia_use_harfbuzz=true skia_pdf_subset_harfbuzz=true skia_use_system_freetype2=false skia_use_system_harfbuzz=false target_cpu=\"x64\" extra_cflags_cc=[\"-frtti\"] cxx=\"g++-9\""
+ninja -C out/Release-x64 skia modules
+
```
Comment (tonsky) [inline script/skia_linux.sh:9]: Better call this file `build_skia_linux.sh`

File: examples/lwjgl/script/build.sh (hunk 1 above)
Comment (olonho) [reply examples/lwjgl/script/build.sh:1]: https://stackoverflow.com/questions/53582831/zsh-on-linux-doesnt-recognize-valid-options-in-a-shell-script

**Comment by tonsky** [comment]
Merged in 337d95639a5da1e2614d63a1d56b0aecfe1846e0

**Pull request closed without merging**